tests/swagger_generation/test_marshmallow_to_swagger.py

```diff
@@ -93,6 +93,27 @@ class Foo(m.Schema):
             }
         )
 
+    def test_dump_only_read_only(self):
```

Comment:
I think you can save some lines and move this to `test_primitive_types`:

Add an item like this to the list:
```
(m.fields.Integer(dump_only=True), {'type': 'integer', 'readOnly': True}),
```

Reply:
Good call, thanks. Done in the latest revision.